src/components/token/Tokens.js

```diff
@@ -213,32 +216,40 @@ class Tokens extends React.Component {
 
     render() {
         return (
-            !shouldRenderCustomTokens ? null :
-                <div className="w-100">
-                    <div className='col-12'>
-                        <h1>{this.props.title}</h1>
-                    </div>
-                    <TokenSearchField
-                        onSearchButtonClicked={this.onSearchButtonClicked}
-                        onSearchTextChanged={this.onSearchTextChanged}
-                        searchText={this.state.searchText}
-                        onSearchTextKeyUp={this.onSearchTextKeyUp}
-                        isSearchLoading={this.state.isSearchLoading}
-                        loading={this.state.loading}
-                    />
-                    <TokensTable
-                        tokens={this.state.tokens}
-                        hasBefore={this.state.hasBefore}
-                        hasAfter={this.state.hasAfter}
-                        onNextPageClicked={this.nextPageClicked}
-                        onPreviousPageClicked={this.previousPageClicked}
-                        loading={this.state.loading}
-                        sortBy={this.state.sortBy}
-                        order={this.state.order}
-                        tableHeaderClicked={this.tableHeaderClicked}
-                        calculatingPage={this.state.calculatingPage}
-                    />
+            <div className="w-100">
+                <div className='col-12'>
+                    <h1>{this.props.title}</h1>
                 </div>
+                {
+                    this.state.maintenanceMode ?
```

Comment:
This ternary inside a ternary is a bit confusing, why not have a separate function to render this part and we can do this conditional rendering with 2 `if`s?